aws/defaults/handlers.go

```diff
@@ -182,6 +183,35 @@ var ValidateResponseHandler = aws.NamedHandler{
 		}
 	}}
 
+// RequestInvocationIDHeaderHandler sets the invocation id header for request
+// tracking across attempts.
+var RequestInvocationIDHeaderHandler = aws.NamedHandler{
+	Name: "core.RequestInvocationIDHeaderHandler",
```

Comment:
Should `Name` for a NamedHandler be an exported constant? In this case, should the `core.RequestInvocationIDHeaderHandler` be exported constant of defaults package?

This would help swap the handlers in case we need to.

Reply:
Since the variable is already exported the general practice has been doing `RequestInvocationIDHeaderHandler.Name` to get the named handler. Having a separate constant duplicates this, and doesn't add a lot of value.